### ISSUE TITLE
discuss.mathdotnet.com is down

### ISSUE THREAD

**Issue by xlash123** [mathnet/mathnet.github.io#1]
Accessing the site https://discuss.mathdotnet.com/ results in a timed-out error.

**Comment by cdrnet** [comment]
Yes, unfortunately I had to take it down due to technical problems, and have not yet found time to revive it. I recommend to use GitHub discussions instead for now.
